.travis.yml

```diff
@@ -24,8 +27,8 @@ jobs:
       before_script: skip
       script: /bin/bash ./bin/lint-commits.sh
       after_success: skip
-      node_js:
-        - "8"
+      os:
```

Comment:
any reason why `osx` is not in the list?

Reply:
Commit message linting only needs to happen once (on any platform) so for this stage we explicitly specify linux. Not specifying any will trigger commit message linting on both platforms which is redundant.